ArduCopter/mode_acro.cpp

```diff
@@ -10,8 +10,8 @@
 void ModeAcro::run()
 {
     // convert the input to the desired body frame rate
-    float target_roll, target_pitch, target_yaw;
-    get_pilot_desired_angle_rates(channel_roll->norm_input_dz(), channel_pitch->norm_input_dz(), channel_yaw->norm_input_dz(), target_roll, target_pitch, target_yaw);
+    float target_roll_cd, target_pitch_cd, target_yaw_cd;
+    get_pilot_desired_rates_cds(channel_roll->norm_input_dz(), channel_pitch->norm_input_dz(), channel_yaw->norm_input_dz(), target_roll_cd, target_pitch_cd, target_yaw_cd);
```

Comment:
Kind of wondering why you removed the "angle" from the names

Reply:
after a quick discussion with Leonard I think losing the "angle" is actually an improvement.  but he will add an "s" at the end.

Reply:
having the angle is inconsistent with what we do in the attitude controller and the name is getting quiet long with the units.